kythe/java/com/google/devtools/kythe/extractors/java/JavaCompilationUnitExtractor.java

```diff
@@ -294,10 +294,12 @@ public CompilationDescription extract(
     return new CompilationDescription(compilationUnit, fileContents);
   }
 
-  /** Returns a new list with the same options except any destination directory options. */
+  /**
+   * Returns a new list with the same options except header/source destination directory options.
+   */
   private static List<String> removeDestDirOptions(Iterable<String> options) {
     return JavacOptionsUtils.removeOptions(
-        Lists.newArrayList(options), EnumSet.of(Option.D, Option.S, Option.H));
+        Lists.newArrayList(options), EnumSet.of(Option.S, Option.H));
```

Comment:
Is the removal and replacement of this option on line 930 still acceptable?

Reply:
Yes; that is done as part of the extraction and (AFAIK) is done so that any generated classes don't clobber the actual compiler output.